lib/AST/ASTPrinter.cpp

```diff
@@ -5455,23 +5647,34 @@ void PrintAST::visitDestructureTupleExpr(DestructureTupleExpr *expr) {
 }
 
 void PrintAST::visitDynamicMemberRefExpr(DynamicMemberRefExpr *expr) {
+  visit(expr->getBase());
+  Printer << ".";
+  Printer << expr->getMember().getDecl()->getName();
 }
 
 void PrintAST::visitDynamicSubscriptExpr(DynamicSubscriptExpr *expr) {
+  visit(expr->getBase());
+  printArgumentList(expr->getArgs()->getOriginalArgs(), true);
+  // forSubscript - is `false` by default
```

Comment:
An inline comment would be better for these e.g
```suggestion
  printArgumentList(expr->getArgs()->getOriginalArgs(), /*forSubscript*/ true);
```